README.md

```diff
@@ -55,7 +55,7 @@ Avro Data Source for Spark supports reading and writing of Avro data from Spark
 
 - **Automatic schema conversion:** It supports most conversions between Spark SQL and Avro records, making Avro a first-class citizen in Spark.
 - **Partitioning:** This library allows developers to easily read and write partitioned data
-witout any extra configuration. Just pass the columns you want to partition on, just like you would for Parquet.
```

Comment:
I don't think this is a valid change since this word a continuation of the previous line's sentence rather than the start of a new sentence. The other line was fixed already, so I'm going to close this PR.